### PR TITLE
Events: Slack logger to debug extracted events

### PR DESCRIPTION
**Pull request by PopDaph**
The goal of this PR is to allow Dust team to easily interact with the Event project.
When defining an EventSchema, we can set a new field debug to True.
If set to True, then when we extract an event we print it here: https://dust4ai.slack.com/archives/C05G9QH1A06/p1689091487639859

Tasks:
- [x] Create new field debug for shemas
- [x] Create a Slack app (super simple, just a bot)
- [x] Create a basic Slack logger lib
- [x] Update the extract event hook to log on slack if the event is attached to a schema marked as debug = True

### CODE REVIEW

File: front/lib/extract_events.ts [hunk 1]
```diff
@@ -149,3 +154,44 @@ async function _runExtractEventApp(
...
+  schema: EventSchema
+): Promise<void> {
+  if (event.eventSchemaId !== schema.id) {
+    logger.error(
+      { event, schema },
+      "[Extract Event] Event schema does not match event."
+    );
+    return;
+  }
+  if (schema.debug !== true) {
```
Comment (PopDaph) [inline front/lib/extract_events.ts:175]: Ceinture et bretelles 😅

**Review by spolu — DISMISSED** [review]
LGTM

left a small suggestion but feel free to ship as is 👍

**Comment by PopDaph** [comment]
@spolu I forced refresh the page and cannot see any suggestion, did you submit the review comments? 🙈 
Thanks for the quick review!

**Comment by spolu** [comment]
Can’t see them. Was just about adding “debug” and maybe “source_url” if we have it to the logger.info for the event creation 👍

**Comment by PopDaph** [comment]
We don't have the source url but it's a great suggestion. I will do it in my next PR as I need to edit the params of the post upsert hooks (not complex but not a one liner, so it deserves its own PR) 😄

**Pull request merged**